app/src/main/java/com/sopt/now/compose/data/module/ApiFactory.kt

```diff
@@ -0,0 +1,42 @@
+package com.sopt.now.compose.data.module
+
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import com.sopt.now.compose.BuildConfig
+import com.sopt.now.compose.data.network.AuthService
+import com.sopt.now.compose.data.network.FollowerService
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+
+
+object ApiFactory {
+    private const val BASE_URL: String = BuildConfig.AUTH_BASE_URL
+    private const val FOLLOWER_URL: String = BuildConfig.FOLLOWER_URL
+
+    private val interceptorClient = OkHttpClient().newBuilder()
+        .addInterceptor(AuthInterceptor()).build()
+
+    val baseRetrofit: Retrofit by lazy {
+        Retrofit.Builder()
+            .baseUrl(BASE_URL)
+            .client(interceptorClient)
+            .addConverterFactory(Json.asConverterFactory("application/json".toMediaType()))
+            .build()
+    }
+
+    val followerRetrofit: Retrofit by lazy {
+        Retrofit.Builder()
+            .baseUrl(FOLLOWER_URL)
+            .addConverterFactory(Json.asConverterFactory("application/json".toMediaType()))
+            .build()
+    }
+
+    inline fun <reified T> createBase(): T = baseRetrofit.create(T::class.java)
+    inline fun <reified T> createFollower(): T = followerRetrofit.create(T::class.java)
+}
```

Comment:
```suggestion
    inline fun <reified T> createFollower(): T = followerRetrofit.create()
}
```

클래스 타입을 매개변수로 넘기지 않아도 된답니다 💪

Reply:
reified으로 컴파일 타임에 타입 정보를 유지해줘서 가능한 거군욥..! 처음 알았습니다 ㅜㅜ 배워갑니다 🤩